tests/unit/cms/classes/CodeParserTest.php

```diff
@@ -1,5 +1,6 @@
 <?php
 
+use Cms\Classes\LayoutCode;
```

Comment:
I'd prefer the `use` statements were in order

Reply:
Didn't understand that - in lexicographical order?